### PR TITLE
📚 Added 0.166.0 changelog

### PR DESCRIPTION
**Pull request by that-one-tom**

### CODE REVIEW

File: docs/reference/changelog.md [hunk 1]
```diff
@@ -2,6 +2,39 @@
...
+
+### New nodes ✨
+
+<br />
+<Changelog node="n8n-nodes-base.odoo" title="Odoo" />
+
+### Enhanced nodes 🚀
+
+<br />
+<Changelog node="n8n-nodes-base.function" title="Function:" text="Added support for items without a json key."/>
```
Comment (StarfallProjects) [inline docs/reference/changelog.md:18]:
```suggestion
<Changelog node="n8n-nodes-base.function" title="Function:" text="Added support for items without a JSON key."/>
```

File: docs/reference/changelog.md [hunk 2]
```diff
@@ -2,6 +2,39 @@
...
+
+### Core functionality ⚙️
+
+- Added new environment variable `N8N_HIRING_BANNER_ENABLED` to enable/disable the hiring banner.
+- Fixed a bug preventing keyboard shortcuts from working as expected.
```
Comment (StarfallProjects) [inline docs/reference/changelog.md:23]: Should these be under bug fixes?

**Review by StarfallProjects — CHANGES_REQUESTED** [review]

File: docs/reference/changelog.md (hunk 2 above)
Comment (that-one-tom) [reply docs/reference/changelog.md:23]: Afaik the bug fix section was only ever for node-specific bug fixes using the `<Changelog>` tag

Comment (StarfallProjects) [reply docs/reference/changelog.md:23]: Ah ok. We probably ought to retitle the bugfixes section to "Node bugfixes" then? As it looks weird to have a load of bugfixes, followed by a bugfixes section

**Pull request merged**